superset/databases/api.py

```diff
@@ -1159,7 +1159,7 @@ def select_star(
         self.incr_stats("init", self.select_star.__name__)
         try:
             result = database.select_star(
-                Table(table_name, schema_name),
+                Table(table_name, schema_name, database.get_default_catalog()),
```

Comment:
Yeah, but I I couldn't find any API calls to this endpoint — it seems like we return this in the database request now, instead of calling `/select_star/`, which is why I didn't change the code to pass a parameter. I think we should keep this endpoint until the next major version and then get rid of it.

Reply:
gotcha! thank you